src/gui/src/settings/options-window.cpp

```diff
@@ -283,6 +287,16 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->comboActionButtons->clear();
+	ui->comboActionButtons->addItem("All", ZoomWindow::ButtonVisibility::All);
+	ui->comboActionButtons->addItem("None", ZoomWindow::ButtonVisibility::None);
+	ui->comboActionButtons->addItem("Favorites", ZoomWindow::ButtonVisibility::Favorites);
+	ui->comboActionButtons->addItem("NonFavorites", ZoomWindow::ButtonVisibility::NonFavorites);
+	ui->comboActionButtons->setCurrentIndex(settings->value("actionButtons", "All").toInt());
```

Comment:
When creating new settings, try to keep them in their setting  group if possible. Here, it would be "Zoom/actionButtons". This allows the setting file to be more readable later. For ini files for example, it will put them in a  `[Zoom]` section together with other image  window-related settings.